storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3114,8 +3114,17 @@ class Rdb_snapshot_status : public Rdb_tx_list_walker {
 
     /* extract table name and index names using the index id */
     std::string table_name = ddl_manager.safe_get_table_name(gl_index_id);
+    if (table_name.empty()) {
+      table_name =
+          "NOT FOUND; INDEX_ID: " + std::to_string(gl_index_id.index_id);
+    }
     auto kd = ddl_manager.safe_find(gl_index_id);
-    std::string idx_name = kd->get_name();
+    std::string idx_name;
+    if (kd == nullptr) {
+      idx_name = "NOT FOUND; INDEX_ID: " + std::to_string(gl_index_id.index_id);
+    } else {
+      idx_name = kd->get_name();
+    }
```

Comment:
You can replace lines 3122-3127 with a more succinct
`std::string idx_name = kd ? kd->get_name() : "NOT FOUND; INDEX_ID: " + std::to_string(gl_index_id.index_id);`